myindex.html

```diff
@@ -0,0 +1,380 @@
+                <p class="articlesPtags ">Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur
+                    erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. Etiam mauris tortor, pharetra […]</p>
+            </div>
+        </div>
+
+        <div class="col-2 ">
+
+            <div class="articlesImagesText ">
+                <h2 class="postedH2 ">POSTED ON <span class="red ">AUGUST 19, 2013</span> BY <span class="red ">ADMIN</span></h2>
+                <p class="articlesPtags ">Donec sed dolor eu augue dignissim pellentesque. Fusce semper tortor ornare, luctus sem in, dictum leo. Ut at congue quam! Maecenas luctus interdum odio, eget volutpat velit hendrerit eu? Suspendisse at magna ut sem blandit hendrerit non
+                    ut magna. Cras quis porta dolor. Aliquam erat volutpat. Phasellus fringilla feugiat libero nec volutpat. Maecenas fermentum bibendum […]</p>
+            </div>
+
+        </div>
+
+        <div class="col-2 ">
+
+            <div class="articlesImagesText ">
+                <h2 class="postedH2 ">POSTED ON <span class="red ">AUGUST 19, 2013</span> BY <span class="red ">ADMIN</span></h2>
+                <p class="articlesPtags ">Quisque sit amet sagittis justo. Suspendisse ornare, lorem estt viverra odio; tristique tristique diam tellus id tellus. In consectetur aliquet tristique. Curabitur imperdiet leo in nisl consectetur mattis. Quisque bibendum, mauris vitae
+                    suscipit ornare; est nibh elementum mauris, eu malesuada nunc magna at justo. Proin blandit id tellus non pellentesque. Mauris […]</p>
+            </div>
+        </div>
+
+        <div class="col-2 ">
+
+            <div class="articlesImagesText ">
+                <h2 class="postedH2 ">POSTED ON <span class="red ">AUGUST 19, 2013</span> BY <span class="red ">ADMIN</span></h2>
+                <p class="articlesPtags ">Aenean quis rhoncus nisl, eget sollicitudin ipsum. Curabitur a elementum purus. Quisque vitae dui at arcu porttitor laoreet ac vitae nibh. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Nulla
+                    facilisi. Nulla egestas at nunc at volutpat. Nullam sodales at diam vel tempor. Sed pretium, leo vel pellentesque dictum, ligula […]</p>
+            </div>
+        </div>
+
+        <div class="col-2 "></div>
+    </div>
+
+    </div>
+
+
+    <!-- Footer -->
+    <hr class="bhr"></hr>
+    <div class="row footerContainer">
+        <div class="insiderow">
+
+            <div class="row row-cols-1 row-cols-md-4">
+                <div class="col mb-4">
+                    <div class="card h-100">
```

Comment:
For what reason is "h-100" applied? Due to this class, in the footer, some cards extend the footer height.